welcome/oke_about.adoc

```diff
@@ -290,7 +290,7 @@ s| Feature s| {oke} s| {product-title} s| Operator name
 | Helm | Included | Included | N/A
 | User Workload Monitoring | Not Included | Included | N/A
 | Cost Management SaaS Service | Included | Included | Cost Management Metrics Operator
-| Platform Logging | Not Included | Included | Red Hat OpenShift Logging Operator
+| Platform Logging | Included | Included | Red Hat OpenShift Logging Operator
```

Comment:
I don't believe this is correct. If I have followed all the threads on the question correctly, we have confirmed that from a technical standpoint the Red Hat OpenShift Logging Operator aka the ClusterLogging Operator can run on OKE, and the CLO provides the ClusterLogForwarder API. So it can theoretically be used exclusively to forward logs. However, the that is only one component of logging, and the ClusterLogging Operator on its own is not quite 'platform logging'.